docs/version0.48.md

```diff
@@ -42,6 +43,14 @@ From release 0.25.0 onwards, `zlibNX` hardware-accelerated data compression and
 
 For more information, see [`-XX:[+|-]UseZlibNX`](xxusezlibnx.md).
 
+### ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) JDWP support on Checkpoint/Restore In Userspace (CRIU) restore is enabled
+
+You can use the options that enable the JDWP support both on CRIU pre-checkpoint, and on restore as well.
+
+Also, a new parameter `suspendOnRestore` for the `Xrunjdwp` option is added to control the suspension of the target VM application on restore. This option is specific to OpenJ9. You can use the `suspendOnRestore=n` setting to prevent the suspension of the target application regardless of whether the debugging was enabled pre-checkpoint or on restore through the restore option file.
```

Comment:
"regardless of whether the debugging was enabled pre-checkpoint or on restore through the restore option file." this part should be removed for this release